system/settings/settings.xml

```diff
@@ -2383,6 +2383,37 @@
           </constraints>
           <control type="spinner" format="integer" />
         </setting>
+        <setting id="nfs.enableids" type="boolean" label="37053" help="37054">
+          <level>2</level>
+          <default>false</default>
+          <control type="toggle" />
+        </setting>
+        <setting id="nfs.uid" type="integer" label="37055" help="37056" parent="nfs.enableids">
+          <level>3</level>
+          <default>1000</default>
+          <constraints>
+            <minimum>1000</minimum>
+            <step>1</step>
+            <maximum>60000</maximum>
```

Comment:
although this is the default max gid value, this can be adjusted, so we shouldn't enforce a hard limit here.